mysql-test/r/percona_heap_var.result

```diff
@@ -192,3 +192,50 @@ count(*)
 10001
 set @@session.max_heap_table_size=default;
 drop table t1;
+#
+# Bug 1731483: MEMORY storage engine incorrectly allows BLOB columns before indexed columns
+#
+CREATE TABLE t1 (a BLOB, b INT PRIMARY KEY) ENGINE=MEMORY;
+ERROR 42000: The used table type doesn't support BLOB/TEXT columns
```

Comment:
Could error message be more specific ? From this I would assume it is not possible at all to create BLOB/TEXT columns inside memory table

Reply:
I have reused one of the existing messages, which is also used in the feature patch elsewhere. IMHO we should not add new error codes to 5.5 at this point